lib/volabit/api/accounts.rb

```diff
@@ -0,0 +1,25 @@
+
+# API endpoints for user account management.
+module Accounts
+  # Creates a new user.
+  # @note Special permissions are required to perform this action.
+  def create_account(acceptance:, email:, pass: '')
+    post_to_resource 'api/v1/users', {
+      accepts_terms_of_service: acceptance,
+      user: {
+        email: email, password: pass
+      }
+    }
+  end
+
+  # Get the account information from the user that authorized the app.
+  def account_details
```

Comment:
Added an alias so the user information can be obtained with `volabit_client.me` or `volabit_client.user_data`